### PR TITLE
Cleanuptests

### PR DESCRIPTION
**Pull request by oscargus**
Refactored some of the cleanup actions and added tests. The tests could be more complete, but better than nothing.

### CODE REVIEW

File: src/main/java/net/sf/jabref/logic/cleanup/LatexCleanup.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+package net.sf.jabref.logic.cleanup;
+
+import net.sf.jabref.model.entry.BibtexEntry;
+
+public class LatexCleanup {
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/logic/cleanup/LatexCleanup.java]: Please try to create a formatter without any dependencies to a BibtexEntry. The setting of the new value should be done in the `Cleanup` class whereas the formatter should also implement the `Formatter` interface. Have a look at other formatters already created, e.g., PageNumberFormatter.

Comment (oscargus) [inline src/main/java/net/sf/jabref/logic/cleanup/LatexCleanup.java]: Oh, quite a few of these things were already done... I'll get back with an updated PR. (I looked at PageNumberFormatter, but clearly not good enough...)

Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/logic/cleanup/LatexCleanup.java]: You might also have a look at #305

Comment (oscargus) [inline src/main/java/net/sf/jabref/logic/cleanup/LatexCleanup.java]: Yes, that PR changed things a bit! :+1:

**Comment by oscargus** [comment]
Replaced with #440

**Pull request closed without merging**